src/GraphQL/QueryFieldConfigGenerator/ImageGallery.php

```diff
@@ -55,7 +55,7 @@ public function getGraphQlFieldConfig($attribute, Data $fieldDefinition, $class
      */
     public function getFieldType(Data $fieldDefinition, $class = null, $container = null)
     {
-        $hotspotType = $this->getGraphQlService()->getTypeDefinition("hotspot");
+        $hotspotType = $this->getGraphQlService()->getTypeDefinition(Hotspotimage::TYPE);
```

Comment:
Don't know how this appeared here when the [change has already been merged](https://github.com/pimcore/data-hub/blob/master/src/GraphQL/QueryFieldConfigGenerator/ImageGallery.php)

Reply:
don't worry ;-)